docker/demo-lammps-mpi/Dockerfile

```diff
@@ -25,7 +25,15 @@ RUN git clone --depth 1 --branch stable_29Sep2021_update2 https://github.com/lam
     cmake ../cmake -D PKG_REAXFF=yes -D BUILD_MPI=yes -D PKG_OPT=yes -D FFT=FFTW3 && \
     make -j ${build_jobs} && \
     make install
-  
+
+# Create a home to put the examples in, and
+# ensure lmp executable is on path for flux user
+# And anyone can interact with lammps examples
+RUN useradd -ms /bin/bash -u 1234 flux && \
+    mv /opt/lammps/examples /home/flux/examples && \
+    chown -R 1234 /home/flux/examples && \
+    cp /root/.local/bin/lmp /usr/local/bin/lmp
```

Comment:
Will copying that clear the setuid bit?
The imp should be 
```
-rwsr-xr-x 1 root root  462976 Sep  8 16:45 flux-imp
```

Reply:
I think flux-imp is in a different tool and spot right? I saw that one in `/usr/libexec/flux` and this one isn't imp, but it's lmp (L vs. I) installed with lammps. I'm totally new to both so sorry if I got that wrong!

Reply:
My mistake, sorry!  Duh, and no `flux-` prefix.

Reply:
You know, I did the same thing! I was confusing my imps and lmls :laughing: So totally understandable!